tests/system/provisioning/vars/wazuh_agent.yml

```diff
@@ -10,3 +10,18 @@ windows_md5: "e4623e7cd3f2fc6ac8a313cbdd7c3cba"
 windows_repo: https://packages.wazuh.com/3.x/windows/
 windows_install_dir: C:\Program Files\ossec-agent\
 windows_install_dir_x86: C:\Program Files (x86)\ossec-agent\
+
+
+### Custom packages installation
+
+# Linux
+
+wazuh_custom_packages_installation_agent_enabled: false
```

Comment:
Agent is false but manager is true, why?.

Reply:
Done in 1d31a131b7e3dbabaafdbe6b314a78889da5619f